lib/Reader/readerir.cpp

```diff
@@ -887,6 +887,89 @@ void GenIR::copyStruct(Type *StructTy, Value *DestinationAddress,
         Alignment, IsVolatile);
 }
 
+void GenIR::copyStruct(CORINFO_CLASS_HANDLE Class, IRNode *Dst, IRNode *Src,
+                       ReaderAlignType Alignment, bool IsVolatile,
+                       bool IsUnchecked) {
+  if (classHasGCPointers(Class)) {
+    GCLayout *RuntimeGCInfo = getClassGCLayout(Class);
+    const uint32_t PointerSize = getPointerByteSize();
```

Comment:
I considered that but felt that the code would be more complicated with nested struct processing. If you strongly prefer that I not use GCLayout, I'd like to change that separately later.
Also, I think now that perhaps I can simplify the code I have here and not try to figure out the exact type of the gc pointer and just use object.